tests/e2e/tests/default.setup.js

```diff
@@ -21,3 +21,19 @@ setup( 'Disable legacy checkout experience', async ( { browser } ) => {
 		page.getByTestId( 'legacy-checkout-experience-checkbox' )
 	).not.toBeChecked();
 } );
+
+setup( 'Disable Link by Stripe', async ( { browser } ) => {
```

Comment:
Link is enabled by default; we should not be disabling it globally... if needed for a specific test, we should disable it in that particular test.

Reply:
@diegocurbelo 👍 Removed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3694/commits/76ceb72da62794305c91ea9296f9ac9846a4a5f6. 

Thought to disable it as I was slightly concerned it may be polluting Link with random email accounts, but I guess it should be fine since we are on test mode anyways.